src/pudl/package_data/ferc1/dbf_to_xbrl.csv

```diff
@@ -3754,13 +3754,13 @@ f1_unrcvrd_cost,46,1994,Line 46,,,,
 f1_unrcvrd_cost,47,1994,Line 47,,,,
 f1_unrcvrd_cost,48,1994,Line 48,,,,
 f1_unrcvrd_cost,49,1994,TOTAL,,,,
-f1_utltyplnt_smmry,1,1994,Total,,,,
-f1_utltyplnt_smmry,2,1994,Electric,,,,
-f1_utltyplnt_smmry,3,1994,Gas,,,,
-f1_utltyplnt_smmry,4,1994,Other (Specify) 1,,,,
-f1_utltyplnt_smmry,5,1994,Other (Specify) 2,,,,
-f1_utltyplnt_smmry,6,1994,Other (Specify) 3,,,,
-f1_utltyplnt_smmry,7,1994,Common,,,,
+f1_utltyplnt_smmry,1,1994,Total,,total,True,
+f1_utltyplnt_smmry,2,1994,Electric,,ferc:ElectricUtilityMember,,
+f1_utltyplnt_smmry,3,1994,Gas,,ferc:GasUtilityMember,,
+f1_utltyplnt_smmry,4,1994,Other (Specify) 1,,ferc:OtherUtilityMember,,
+f1_utltyplnt_smmry,5,1994,Other (Specify) 2,,ferc:OtherUtility3Member,,
+f1_utltyplnt_smmry,6,1994,Other (Specify) 3,,ferc:OtherUtility2Member,,
+f1_utltyplnt_smmry,7,1994,Common,,ferc:CommonUtilityMember,,
```

Comment:
these are mapped to the [utility_type_axis](https://xbrlview.ferc.gov/yeti/resources/yeti-gwt/Yeti.jsp#tax~(id~8*v~72)!con~(id~80674)!net~(a~143*l~35)!lang~(code~en)!path~(g~3847*p~0_0_0_0)!rg~(rg~4*p~1)) reported in xbrl. after concatenation, these are renamed using the `categorize_strings` transformer.

Reply:
Does this seem like a good way to work with this kind of categorical value? I've seen them cropping up elsewhere in other tables too. Is it some kind of XBRL categorical specification? If so, I wonder if we can create a dataset-wide mapping, since the same categories show up in multiple tables, and we'd presumably want them all to be renamed identically so they can be compared.

Reply:
it was very easy to deal with once I added these mappings. Since it doesn't look like there is  a very standard format for the dbf data (via some searching through this file), I'd suggest actually waiting to see what it looks like for other tables.

Reply:
Oh I guess there's going to be two different sets of values that have to get categorized -- one from the DBF data and one from the XBRL. I wonder if there's a useful way to use this mapping to identify what all values in the DBF data correspond to the (probably cleaner?) value in the XBRL data.

Agreed we should wait for more context.